apps/coscrad-frontend/src/components/header/header.tsx

```diff
@@ -9,6 +9,7 @@ import { NavMenuContainer } from '../nav-menu/nav-menu-container';
 const StyledAppBar = styled(AppBar)({
     backgroundColor: '#2cb5af',
     position: 'sticky',
```

Comment:
This is a thing? Sounds like things have gotten easier since the last time I had to do that!